sdk/python/v1beta1/kubeflow/katib/utils/utils.py

```diff
@@ -69,8 +73,8 @@ def print_experiment_status(experiment: models.V1beta1Experiment):
                 experiment.status.trial_metrics_unavailable or 0,
             )
         )
-        print(f"Current Optimal Trial:\n {experiment.status.current_optimal_trial}")
-        print(f"Experiment conditions:\n {experiment.status.conditions}")
+        logger.debug(f"Current Optimal Trial:\n {experiment.status.current_optimal_trial}")
+        logger.debug(f"Experiment conditions:\n {experiment.status.conditions}")
```

Comment:
Do we want to keep `print` here similar to get_job_logs API in Training Client: https://github.com/kubeflow/training-operator/blob/master/sdk/python/kubeflow/training/api/training_client.py#L1193 ?
Since when user runs this API, the SDK works like CLI and user wants to get output.
WDYT @droctothorpe @tariq-hasan @kubeflow/wg-automl-leads ?

Reply:
I have reverted the changes to keep `print`.